### DISCUSSION TITLE
Integration with Quadro

### DISCUSSION THREAD

**Discussion by daniel-marechal** [Zettlr/Zettlr#3661] [Ideas]
One of the things I like about Zettlr is the straightforward integration with RMarkdown, as both of them use the .md extension. I enjoy using Zettlr to categorize and organize my files, using the available Zettelkasten tools, and to work with plain text, while I can use the very same md file to do some data analysis with R. Last year Quadro was launched to (I assume) replace RMarkdown, making it easier to work with R alongside other program languages (Python). Unlike RMarkdown, Quadro uses the .qmd  extension, which as of now is not compatible with Zettler - even though it is basically another kind of markdown file. Therefore, I believe that making Zettlr compatible with .qmd extension would be a nice (and maybe not too complicated) improvement.

**Comment by nathanlesage** [comment]
I've added the qmd-extension I think yesterday, so it'll be coming with the next update. Maybe even further integration (using Quarto instead of Pandoc etc.) could be thinkable.

**Comment by EcoFin** [comment]
I too have been experimenting with Quarto.  I haven't had any particular difficulty going back and forth between qmd and md files.  But I am looking forward to trying out the new zettlr qmd extension.  I have just finished setting up an ad hoc system to use zettlr and quarto (and mkdocs, in fact) together on a quick book project.  The salient differences for me are:
1. zettlr's file management
2. quarto's easy, low overhead abiliity to produce websites and
3. quarto's ability to let me easily integrate ipynb files (same for rmarkdown; I just don't do that much R any more)
